backend/db_setup/data.indaba.sql

```diff
@@ -36,7 +36,7 @@ COPY "Essences" (id, "tableName", name, "fileName", "nameField") FROM stdin;
 23	WorflowSteps	WorflowSteps	worflowSteps	title
 20	Groups	Groups	groups	title
 21	Organizations	Organizations	organizations	name
-22	Tasks	Tasks	tasks	title
+22	Tasks	Tasks	tasks	description
```

Comment:
What information would description hold?

Reply:
Honestly nothing. I'm not sure this has value. Open to suggestions.